client/src/plugins/version-info/ReleaseInfo.less

```diff
@@ -17,4 +18,20 @@
       margin-bottom: 0;
     }
   }
+
+  div {
```

Comment:
Why a `div` here? What shall we accomplish? 

My proposal is to make this an actual heading instead, i.e. `h4` to match semantic HTML.